grails-app/views/langSelector/_selector.gsp

```diff
@@ -1,21 +1,13 @@
-<%@ page import="grails.plugins.GrailsPluginManager" %>
-<g:set var="locales" value="${(List<Locale>)locales}" />
-<g:set var="selected" value="${(Locale)selected}" />
+<%@ page import="grails.util.Holders; grails.plugins.GrailsPluginManager" %>
 <g:set var="hasAsset"
```

Comment:
you mean remove it at all and don't use it ?, cause I just set it in this general template and use in child templates.